src/main/kotlin/app/batch/BufferingProcessor.kt

```diff
@@ -11,7 +10,7 @@ import java.io.ByteArrayOutputStream
 
 @Component
 @Qualifier("buffer")
-class BufferingProcessor() : ItemProcessor<EncryptedStream, EncryptedStream> {
```

Comment:
why are the parenthesis removed?

Reply:
It defines an empty primary constructor which is redundant - intellij code clean-up which I run before pushing suggests getting rid of them and other things like unused imports etc.

Reply:
They are not needed in declaring the class, syntax should be to remove them so I think this is ok.